### PR TITLE
Add wiki Link for settings

### LINKED ISSUE #322: Add wiki Link for settings
Add wiki Link for settings

### PR DESCRIPTION
**Pull request by angelozerr**
Add wiki Link for settings

Fixes #322

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
It's not finished, but it's just to give you some idea how to I would like to manage that.

@fbricon @datho7561  please add your comments.

File: package.json [hunk 1]
```diff
@@ -88,22 +112,51 @@
...
+            ]
+          },
+          "markdownDescription": "Allows XML schemas/ DTD to be associated to file name patterns.\n\nExample:\n```json\n[{\n  \"systemId\": \"path/to/file.xsd\",\n  \"pattern\": \"file1.xml\"\n},\n{\n  \"systemId\": \"http://www.w3.org/2001/XMLSchema.xsd\",\n  \"pattern\": \"**/*.xsd\"\n}]\n```",
+          "scope": "window"
+        },
+        "xml.format.enabled": {
           "type": "boolean",
           "default": true,
-          "description": "Should the server log to client output."
+          "markdownDescription": "Enable/disable ability to format document. Default is `true`. See [here](https://github.com/redhat-developer/vscode-xml/wiki/Formatting#xmlformatenabled-) for more information.",
```
Comment (datho7561) [inline package.json:158]: I don't think we need a link here, since the wiki says the same thing as the description.

File: README.md [hunk 2]
```diff
@@ -49,37 +49,37 @@ See the [changelog](CHANGELOG.md) for the latest release. You might also find us
...
+* `xml.server.workDir`: Set a custom folder path for cached XML Schemas. An absolute path is expected, although the `~` prefix (for the user home directory) is supported. Default is `~/.lemminx`.
+* `xml.trace.server` : Trace the communication between VS Code and the XML language server in the Output view. Default is `off`.
+* `xml.logs.client` : Enable/disable logging to the Output view. Default is `true`.
+* `xml.catalogs` : Register XML catalog files. See how to configure [XML catalog with XSD](https://github.com/redhat-developer/vscode-xml/wiki/XMLValidation#xml-catalog-with-xsd) or [XML catalog with DTD](https://github.com/redhat-developer/vscode-xml/wiki/XMLValidation#xml-catalog-with-dtd) for more information.
+* `xml.fileAssociations` : Allows XML schemas/ DTD to be associated to file name patterns.
+* `xml.format.enabled` : Enable/disable ability to format document. Default is `true`. See [here](https://github.com/redhat-developer/vscode-xml/wiki/Formatting#xmlformatenabled-) for more information.
```
Comment (datho7561) [inline README.md:59]: Same comment as for `package.json`

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (angelozerr) [reply package.json]: fixed

File: README.md (hunk 2 above)
Comment (angelozerr) [reply README.md]: fixed

**Review by datho7561 — APPROVED** [review]

**Pull request merged**